sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveQlSuite.scala

```diff
@@ -201,6 +202,31 @@ class HiveQlSuite extends SparkFunSuite with BeforeAndAfterAll {
     assert(plan.children.head.asInstanceOf[Generate].generator.isInstanceOf[JsonTuple])
   }
 
+  test("transform query spec") {
+    val plan1 = parser.parsePlan("select transform(a, b) using 'func' as c, d from e")
```

Comment:
What is the reason for changing the test? Is HiveQlSuite written in a different style?